component/topolvm_alerts.yaml

```diff
@@ -0,0 +1,31 @@
+groups:
+  - name: topolvm-alert.rules
+    rules:
+      - alert: TopoLVMCriticallyFull
```

Comment:
I would prefer having "Full" as real full. Block devices can be filled 100% without issues. So I would prefer to use an AlmostFull or Critical for 95% used and Full for 100%. I think there is a difference in emergency you have to react on.

Reply:
Misinterpret the alert name. This is affecting the volume group and not the logical volume. Since this alert is used for kind of a capacity management, severity_level of warning might be ok as well.